elements/pfe-autocomplete/src/pfe-autocomplete.js

```diff
@@ -27,6 +27,74 @@ class PfeAutocomplete extends PFElement {
     return "pfe-autocomplete.scss";
   }
 
+  static get properties() {
+    return {
+      initValue: {
+        title: "Initial Value",
+        type: String,
+        observer: "_initValueChanged"
+      },
+      _initValue: {
+        // @TODO Deprecate property
+        type: String,
+        prefix: false,
+        attr: "init-value",
```

Comment:
Is this in the readme about what options are available in the property defintions?

Reply:
`docs/content/develop/properties.md`

Reply:
"It is now, yeah." said Michael Clayton